src/slider/example/index.ts

```diff
@@ -60,6 +60,7 @@ export class App extends WidgetBase<WidgetProperties> {
 				},
 				step: 1,
 				value: tribbleValue,
+				onChange: this.onTribbleInput,
```

Comment:
Does this mean that both onChange and onInput are required when using this component in certain browsers?

Reply:
(we chatted about this, only for VO + Safari)

Reply:
I need to remember to file a bug report with VoiceOver :P